multicluster/test/integration/crds/acnp-crd.yml

```diff
@@ -0,0 +1,539 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Why we need to copy CRD Yamls?

Reply:
We need one CRD `clusternetworkpolicies.crd.antrea.io` which is used in copy-span ACNP, but we only have a full CRD definitions file, so I copy it. maybe I should use the full CRD definitions. let me refine it.